### PR TITLE
Preciso Bid Adapter :  update on valid request checks

### PR DESCRIPTION
**Pull request by NikhilGopalChennissery**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by ChrisHuie** [comment]
seems similar to #11106

**Comment by NikhilGopalChennissery** [comment]
> seems similar to #11106

Hi @ChrisHuie , this PR (#11161 ) has the latest changes that we have made. Could you please review this PR instead of [#11106](https://github.com/prebid/Prebid.js/pull/11106)? We would like to close the old PR ([#11106](https://github.com/prebid/Prebid.js/pull/11106)).

**Comment by NikhilGopalChennissery** [comment]
Hi @ChrisHuie , we would like to know if there are any issues with the PR for merging. 

Thank you

**Comment by ChrisHuie** [comment]
@jlquaccia can you take a look at this when you get a chance

File: modules/precisoBidAdapter.js [hunk 1]
```diff
@@ -1,99 +1,94 @@
...
+// const AD_URL = 'http://localhost:80/bid_request/openrtb';
 const URL_SYNC = 'https://ck.2trk.info/rtb/user/usersync.aspx?';
 const SUPPORTED_MEDIA_TYPES = [BANNER, NATIVE, VIDEO];
 const GVLID = 874;
 let userId = 'NA';
+let precisoId = 'NA';
+let sharedId = 'NA'
+
+export const storage2 = getStorageManager({ moduleType: MODULE_TYPE_UID, moduleName: BIDDER_CODE });
+export const storage = getStorageManager({ moduleType: MODULE_TYPE_UID, moduleName: 'sharedId' });
```
Comment (ChrisHuie) [inline modules/precisoBidAdapter.js:21]: Will have to ask about this in a JS meeting because never seen sharedId also accessed in a bid adapter.

**Comment by NikhilGopalChennissery** [comment]
Hey @ChrisHuie, is there are any concerns with the PR for merging.

**Comment by ChrisHuie** [comment]
> Hey @ChrisHuie, is there are any concerns with the PR for merging.
> 
> Thank you

I can finish up this review. Jason might be busy with Fledge and other crazy ad world stuff. My only question is around the access of sharedId also? Don't believe I have seen that in an adapter before but think liveintent might do something similar.

**Comment by NikhilGopalChennissery** [comment]
Hi @ChrisHuie, We are using this sharedId for the purpose of usersync. Will it cause any impact if we use sharedId in bid Adapter? We don't have any idea about access the sharedId from bid Adapter. So could you please guide us for further steps.

**Comment by ChrisHuie** [comment]
I assigned myself to review today or over the weekend to get this adapter into prebid

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**